File: STL/stack.cpp
#include <iostream>
#include <stdio.h>
#include <string>
#include <list>
#include <queue>
#include <iterator>
#include <algorithm>
#include <map>
#include<utility>
#include<set>
#include <stack>

using namespace std;

int main()
{
    stack <string> s;

    s.push("Siam");
    s.push("Anik");
    s.push("Foyaz");

    // cout << s.top() << endl;
    // s.pop();
    // cout << s.top() << endl;

    while(!s.empty())
    {
        string x;
        x = s.top();
        cout << x << endl;
        s.pop();
    }


    return 0;
}

File: js/stack.js
class Stack {
    constructor(){
        this.items = [];
    }

    push(element){
        this.items.push(element);
    }

    pop(){
        if(this.items.length === 0){
            return false;
        }
        return this.items.pop();
    }
}

const stack = new Stack();
stack.push(10);
stack.push(20);
console.log(stack.pop());
console.log(stack);